### PR TITLE
Permit newer versions of requests

### PR DESCRIPTION
**Pull request by alex**
Required to fix CVE-2023-32681

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -3,7 +3,7 @@ cement==2.8.2
 colorama>=0.2.5,<0.4.4 # use the same range that 'docker-compose' uses
 pathspec==0.10.1
 python-dateutil>=2.1,<3.0.0 # use the same range that 'botocore' uses
-requests>=2.20.1,<=2.26
+requests>=2.20.1,<=2.31
```
Comment (benxiao) [inline requirements.txt:6]: should be `requests >= 2.31`. so people don't get the vulnerable versions.

**Comment by alex** [comment]
There's no need to rebase. Your commit replaces this one.

Please note that your commit also checks in the `venv/` directory, which I doubt is intentional: https://github.com/aws/aws-elastic-beanstalk-cli/commit/7d2ba9837b717a71061c1ad075a2ae375b2b26f8

**Comment by NihalM99** [comment]
Should I publish another commit after removing the venv directory from my local ?

**Comment by jl-applied** [comment]
@alex can we get a release for this? Thanks

**Comment by alex** [comment]
I do not work for AWS, I am not a maintainer of this package, I cannot answer questions about a release or how to handle a bad commit.

These questions should be directed to the maintainers/AWS.

**Pull request closed without merging**